### PR TITLE
Fixup: Add requirements for avocado

### PR DESCRIPTION
**Pull request by sathnaga**
Let's try installing requirement python packages required
for avocado.

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

### CODE REVIEW

File: avocado-setup.py [hunk 1]
```diff
@@ -331,7 +331,7 @@ def install_repo(path):
     :param repo: repository path
     """
     logger.info("Installing repo: %s", path)
-    cmd = "cd %s;make requirements;python setup.py install" % path
+    cmd = "cd %s;make requirements;requirements-selftests;python setup.py install" % path
```
Comment (narasimhan-v) [inline avocado-setup.py]: isn't it `make requirements-selftests` ?

Comment (sathnaga) [reply avocado-setup.py]: yep, that's right, missed it..

**Comment by PraveenPenguin** [comment]
@sathnaga still facing  issue  :( 

```[root@ltczep10-lp6 avocado-fvt-wrapper]# python avocado-setup.py --bootstrap
09:48:17 INFO    : Check for environment
09:48:20 INFO    : install packages via pip interface
09:48:21 INFO    : Creating Avocado Config
09:48:21 INFO    : Uninstalling avocado and autotest from environment
09:48:21 ERROR   : Error in removing package: avocado sh: pip: command not found
09:48:21 ERROR   : Error in removing package: avocado_plugins_vt sh: pip: command not found
09:48:21 ERROR   : Error in removing package: autotest sh: pip: command not found
09:48:21 INFO    : Bootstrapping
09:48:21 INFO    : Updating the repo: avocado in /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/avocado
09:48:21 INFO    : Installing repo: /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/avocado
09:48:22 INFO    : Updating the repo: avocado-vt in /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/avocado-vt
09:48:22 INFO    : Installing repo: /home/workspace/runAvocadoFVTTest/avocado-fvt-wrapper/avocado-vt
09:48:25 INFO    : Bootstrapping vt libvirt
09:48:25 ERROR   : Failed to bootstrap vt libvirt. Error: Failed to load plugin from module "avocado_vt.plugins.vt_list": ImportError('No module named netifaces',) :
Traceback not available
Failed to load plugin from module "avocado_vt.plugins.vt": ImportError('No module named netifaces',) :
Traceback not available
Failed to load plugin from module "avocado_vt.plugins.vt_bootstrap": ImportError('No module named aexpect.utils.genio',) :
Traceback not available
usage: avocado [-h] [-v] [--config [CONFIG_FILE]] [--show [STREAM[:LVL]]] [-s]
               {config,diff,distro,exec-path,list,multiplex,plugins,run,sysinfo,variants}
               ...

Avocado Test Runner

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show program's version number and exit
  --config [CONFIG_FILE]
                        Use custom configuration from a file
  --show [STREAM[:LVL]]
                        List of comma separated builtin logs, or logging
                        streams optionally followed by LEVEL (DEBUG,INFO,...).
                        Builtin streams are: "test": test output; "debug":
                        tracebacks and other debugging info; "app":
                        application output; "early": early logging of other
                        streams, including test (very verbose); "remote":
                        fabric/paramiko debug; "all": all builtin streams;
```

**Comment by PraveenPenguin** [comment]
after installing `make` package it works fine

**Pull request merged**